ci_tests/test_pygoat_findings.py

```diff
@@ -6,7 +6,9 @@
     "pixee:python/add-requests-timeouts",
     "pixee:python/secure-random",
     "pixee:python/sandbox-process-creation",
+    "pixee:python/subprocess-shell-false",
     "pixee:python/django-session-cookie-secure-off",
+    "pixee:python/django-model-without-dunder-str",
```

Comment:
I'm not really sure how to account for this change 😅

Reply:
this is really surprising... I can see pygoat django Model classes that would merit this codemod, but why this wasn't getting triggered before 🤷